app/Domain/Mail/MailManager.php

```diff
@@ -43,13 +43,23 @@ public function sendIntegrationCreatedMail(IntegrationCreatedWithContacts $event
     {
         $integration = $this->integrationRepository->getById($event->id);
 
+        if ($integration->type === IntegrationType::UiTPAS && config(UiTPASConfig::AUTOMATIC_PERMISSIONS_ENABLED->value)) {
```

Comment:
I think the check for the feature flag has no added value and is not even needed.

Reply:
removed the flag